src/impls/create.rs

```diff
@@ -17,6 +17,13 @@ where
             zero: N::zero(),
         }
     }
+    #[must_use]
```

Comment:
It's not obvious why this is marked as `#[must_use]` when `new` is not. In general I am reluctant to add that annotation to this library.

Reply:
i thought I did a single commit PR to add it in `new`. Could you share any specific reasons as to the reluctance?
Anyway if you would like to not use it in this library, what do you feel about adding `#![allow(clippy::must_use)]` to suppress this lint?

Reply:
`#[must_use]` is a very helpful attribute when applied to `Result<T, E>`, because it helps to ensure that we don't accidentally forget to handle errors. It is not a very helpful attribute IMO when applied to a constructor, because the only time people call a constructor is when they want its value. 

Clippy itself seems to agree. See, for example, the ["limitations" section of `clippy::return_self_not_must_use`](https://rust-lang.github.io/rust-clippy/master/index.html#/return_self_not_must_use):

> It would be mostly noise if it was added on constructors for example.

As for specifically allowing `clippy::must_use`, that appears [not to be a lint](https://rust-lang.github.io/rust-clippy/master/index.html#/must_use). I do not particularly mind adding it though, if it turns out to be one.

Reply:
Sure, `#![allow(clippy::must_use_candidate)]` is acceptable--but in a different, single-purpose PR. The name of the game here is tight, focused PRs.

Reply:
understood.